fusion_inputs.json

```diff
@@ -1,16 +1,16 @@
-  "runfusions.wf_fusions_py": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/fusions.py",
-  "runfusions.wf_ROIs": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/ROIs.txt",
-  "runfusions.wf_dnmt3a_bed": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/dnmt3a.bed",
-  "runfusions.wf_gene_ref_bed": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/gene_reference.bed",
-  "runfusions.wf_ref_fa": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/references/all_sequences.fa",
-  "runfusions.wf_ref_fa_fai": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/references/all_sequences.fa.fai",
-  "runfusions.wf_ref_dict": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/references/all_sequences.dict",
-  "runfusions.Mutect2_intervals": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/dnmt3a.interval_list",
-  "runfusions.Mutect2_scatter_count": "10",
-  "runfusions.Mutect2_gatk_docker": "broadinstitute/gatk",
-  "runfusions.Mutect2_gatk_override": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/gatk4261local.jar"
+  "fusions_mutations.wf_fusions_bed": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/for_git/data/regions.bed",
+  "fusions_mutations.wf_full_cram": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/testing/tumor.cram",
+  "fusions_mutations.wf_full_cram_crai": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/testing/tumor.cram.crai",
+  "fusions_mutations.wf_ROIs": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/for_git/data/ROIs.txt",
+  "fusions_mutations.wf_subset_bed": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/for_git/data/subset_exons.bed",
+  "fusions_mutations.wf_gene_ref_bed": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/for_git/data/gene_reference.bed",
+  "fusions_mutations.wf_ref_fa": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/references/all_sequences.fa",
+  "fusions_mutations.wf_ref_fa_fai": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/references/all_sequences.fa.fai",
+  "fusions_mutations.wf_ref_dict": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/mutect2/references/all_sequences.dict",
+  "fusions_mutations.Mutect2_intervals": "/storage1/fs1/timley/Active/aml_ppg/tmp/jonathanztang/breakpoint_reader/terra/for_git/data/subset_exons.interval_list",
+  "fusions_mutations.Mutect2_scatter_count": "10",
+  "fusions_mutations.Mutect2_gatk_docker": "broadinstitute/gatk",
```

Comment:
It would be good to pin this to a specific version of the docker image (e.g. broadinstitute/gatk:4.1.1 or whatever)

Reply:
Pinned to 4.2.6.1 (current latest version)